packages/next/client/link.js

```diff
@@ -144,6 +144,11 @@ class Link extends Component {
       props.href = _Router._rewriteUrlForNextExport(props.href)
     }
 
+    const isCurrent = props.href === router.pathname
```

Comment:
Should this be:

```js
const isCurrent = props.href === router.asPath
```

Reply:
Yeah probably, wasn't aware of the `asPath` property 👍